### PR TITLE
C# 9.0 doc updates

### PR DESCRIPTION
**Pull request by benmccallum**
## Summary

Couple of changes now that C# 9.0 is released.

### CODE REVIEW

File: docs/csharp/language-reference/configure-language-version.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Builds in all subdirectories of the directory containing that file will use the
 
 ## C# language version reference
 
-The following table shows all current C# language versions. Your compiler may not necessarily understand every value if it's older. If you install .NET Core 3.0 or later, then you have access to everything listed.
+The following table shows all current C# language versions. Your compiler may not necessarily understand every value if it's older. If you install the latest .NET SDK, then you have access to everything listed.
```
Comment (benmccallum) [inline docs/csharp/language-reference/configure-language-version.md:78]: Seems kind of pointless to be overly specific here and need to update it all the time.

File: docs/csharp/language-reference/configure-language-version.md [hunk 2]
```diff
@@ -101,7 +101,8 @@ The following table shows all current C# language versions. Your compiler may no
 > 7.1
 > 7.2
 > 7.3
-> 8.0 (default)
+> 8.0
```
Comment (BillWagner) [inline docs/csharp/language-reference/configure-language-version.md:104]: quick nit for the markdownlint error (whitespace removal):

```suggestion
> 8.0
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
This looks great @benmccallum 

If you accept the one white-space suggestion I made, I'll :shipit:.  That will fix the markdownlint error.

Thanks again!

**Comment by benmccallum** [comment]
Whoops, that's what you get for editing in GitHub haha. Fixed and back to you @BillWagner

**Review by BillWagner — APPROVED** [review]

**Pull request merged**